### ISSUE TITLE
support Symbolics.jl >v6.15.1

### ISSUE THREAD

**Issue by oameye** [NonlinearOscillations/HarmonicBalance.jl#279]
[This PR](https://github.com/JuliaSymbolics/Symbolics.jl/pull/1305) in symbolics break HB.jl.

**Comment by oameye** [comment]
```julia
# Taken from Symbolics.jl and reduced to the minimum necessary for the code to work for param and variables.
# https://github.com/JuliaSymbolics/Symbolics.jl/blob/c3f54a60c3b708ed33411a431d040de3a02d5f3d/src/latexify_recipes.jl#L140
# Symbolics._toexpr does \\mathtt{var} for variables
# see https://github.com/JuliaSymbolics/Symbolics.jl/pull/1305
function _toexpr(O)
    if Symbolics.issym(O)
        sym = string(nameof(O))
        return Symbol(replace(sym, Symbolics.NAMESPACE_SEPARATOR => "."))
    end
    !Symbolics.iscall(O) && return O

    op = Symbolics.operation(O)
    args = Symbolics.sorted_arguments(O)

    if (op===(*)) && (args[1] === -1)
        arg_mul = Expr(:call, :(*), _toexpr(args[2:end])...)
        return Expr(:call, :(-), arg_mul)
    end

    if Symbolics.symtype(op) <: Symbolics.FnType
        isempty(args) && return nameof(op)
        return Expr(:call, _toexpr(op), _toexpr(args)...)
    elseif Symbolics.issym(op) &&  Symbolics.symtype(op) <: AbstractArray
        return :(_textbf($(nameof(op))))
    elseif op === identity
        return _toexpr(only(args)) # suppress identity transformations (e.g. "identity(π)" -> "π")
    end
    return Expr(:call, Symbol(op), _toexpr(args)...)
end
_toexpr(O::Num) = _toexpr(O.val)

```

**Comment by oameye** [comment]
solved by #292
